clients/linux/src/account.rs

```diff
@@ -364,19 +365,23 @@ impl Editor {
         let cntr = GtkScrolledWindow::new(None::<&GtkAdjustment>, None::<&GtkAdjustment>);
         cntr.add(&stack);
 
+        let highlighter = LanguageManager::new();
```

Comment:
possible nit: I think a good pattern for constructors is if the variable isn't used for something during initializing, just define in the return value:
```
Self {
    ....
    highlighter: LanguageManager::new(),
    ....
}
```

Reply:
fixed in: 1119c4177b7c3f0e40f73f18da95eaeef2dbd2e0